js/script.js

```diff
@@ -0,0 +1,155 @@
+let data;
+
+// getting the section we need to change
+let section = document.querySelector(".explain");
+
+
+let courses = ["Python", "AWS", "Data Science", "Excel", "Web Development", "JavaScript", "Drawing"]
```

Comment:
data that will not be changed should declared as const 
```suggestion
const courses = ["Python", "AWS", "Data Science", "Excel", "Web Development", "JavaScript", "Drawing"]
```